new_tab/about.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="../css_folder/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <div class="navbarMobile">
+          <ul class="menumo">
+            <li class="grid">
+              <a href="#"><ion-icon name="grid"></ion-icon></a>
+            </li>
+            <li class="menu-nth"></li>
+            <li class="subscribe">Subscribe</li>
+          </ul>
+        </div>
+      </nav>
+
+      <nav class="header1">
+        <ul>
+          <li><ion-icon name="logo-facebook"></ion-icon></li>
+          <li><ion-icon name="logo-twitter"></ion-icon></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </nav>
+      <nav class="header2">
+        <div class="navbarDesktop">
+          <span
+            >GLCon23'<ion-icon id="naveicon" name="aperture-outline"></ion-icon
+          ></span>
+          <ul>
+            <li id="about"><a href="../index.html"> Homepage</a></li>
+            <li><a href="#customBackground">About</a></li>
+            <li><a href="#Location">Location</a></li>
+            <li><a href="#sponsors">Sponsor</a></li>
+          </ul>
+
+          <button class="navbutt">Campaign</button>
+        </div>
+      </nav>
+
+      <nav>
+        <div class="slidingMenu">
+          <div class="closeMenu">
+            <img src="../Images/WindowDisabled.png" alt="WindowDisabled" />
+          </div>
+          <ul>
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="#"> About</a></li>
+            <li><a href="../index.html#programContainer">Program</a></li>
+            <li><a href="../index.html#featuredSpeakers">Speakers</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div id="customBackground" class="customBackground">
+        <div class="aboutPage">
+          <h2>About Us</h2>
+        </div>
+        <p class="aboutPara">
+          The Global Leaders' Conference brings together a community of expert,
+          intellectuals and scholars who make up the Global leaders' fratenity
+          affiliate network in a diffrent country every two years. This year we
+          hope to expand our scope across the entire Microverse reaching out to
+          the globe. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Perferendis consequatur, ratione quod omnis ipsa aperiam quaerat
+          facilis recusandae quas hic saepe dolore a debitis, quos inventore
+          commodi nihil reiciendis consectetur!
+        </p>
+
+        <div class="contactSec">
+          <p>
+            Please contact us per Email for any further questions. We would
+            really be glad to assist you! <br />

```

Comment:
## Desktop Version

-  [ ] As I mentioned above :point_up_2: please stick to the Behance design but the heading is missing and also the background image is missing and also the spacing and alignments issues are there and also font size and font family are not the same as the Behance design so kindly fix these all issues :+1:  check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 15-58-36](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/f6f1d2b9-cd78-4528-9ff5-bdf5947e10fd)

**Behance Design**
![Screenshot from 2023-07-07 15-58-20](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/6294bedd-e470-4e57-a917-45e788c78c3d)

## Mobile Version

-  [ ] As I mentioned above :point_up_2: please stick to the Behance design but the background image is missing and also the heading is not match with Behance design and also the font size and font family is not matching with Behance design  so kindly fix all these issues as well :+1: check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 16-02-03](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/d059c410-bb29-40d9-b6bf-41b65efdfb97)

**Behance Design**
![Screenshot from 2023-07-07 16-02-42](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/275ce13a-96b9-4516-bca3-1b206ca85d54)

- [ ] Also, if you check your design there are some spacing and alignment issues so kindly fix this issue as well :+1: